test_agasc1p7.py

```diff
@@ -38,7 +38,7 @@
 ! the AXAF Guide and Acquisition Star Catalog (AGASC) version 1.7
 ! V1.7 changes values for MAG_ACA, MAG_ACA_ERR, RSV1, RSV2 and RSV3""".split('\n')
 
-h5_fits = Table.read('agasc1p7_from_fits.h5', path='data')
+h5_fits = Table.read('agasc1p7_indexed_from_fits.h5', path='data')
```

Comment:
File name mismatch here 'agasc1p7_from_fits_indexed.h5' used above.  Looks good to me besides that.

Reply:
Fixed